pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -169,7 +169,7 @@ func (i *Gatherer) Gather(ctx context.Context, recorder record.Interface) error
 // See: docs/insights-archive-sample/config/pdbs
 func GatherPodDisruptionBudgets(i *Gatherer) func() ([]record.Record, []error) {
 	return func() ([]record.Record, []error) {
-		pdbs, err := i.policyClient.PodDisruptionBudgets("").List(i.ctx, metav1.ListOptions{Limit: gatherPodDisruptionBudgetLimit,})
+		pdbs, err := i.policyClient.PodDisruptionBudgets("").List(i.ctx, metav1.ListOptions{Limit: gatherPodDisruptionBudgetLimit})
```

Comment:
I have the impression that I've already seen these changes but the other way around. Is this the `go fmt` standard? Otherwise LGTM

Reply:
It was formatted with `go fmt`. I guess that the recommendation for ending with `,` is only if you have multiple lines, then the last one should end with `,`.